microsoft-edge/webview2/how-to/javascript.md

```diff
@@ -89,59 +89,59 @@ To solve the problem, create a separate JavaScript file with your code, and then
 
 
 <!-- ====================================================================== -->
-## Scenario: Removing drag-and-drop functionality
+## Scenario: Removing drag-and-drop function
```

Comment:
I think I would leave this as "functionality". "Function" has a specific meaning (like a single function that can be called) whereas Drag and Drop and Context Menu are more a feature or functionality of the control. (This also applies down below)

Reply:
fixed